Swiftfin/Views/AdminDashboardView/ServerUserPermissionsView/Components/Sections/SessionsSection.swift

```diff
@@ -0,0 +1,193 @@
+                    tempLoginAttempts = policy.loginAttemptsBeforeLockout
+                }
+
+                if isCustomLoginFailurePolicy {
+                    MaxFailedLoginsButton()
+                }
+
+            } header: {
+                Text(L10n.sessions)
+            } footer: {
+                VStack(alignment: .leading) {
+                    Text(L10n.maximumFailedLoginPolicyDescription)
+                    LearnMoreButton(L10n.maximumFailedLoginPolicy) {
+                        TextPair(
+                            title: L10n.unlimited,
+                            subtitle: L10n.unlimitedFailedLoginDescription
+                        )
+                        TextPair(
+                            title: L10n.default,
+                            subtitle: L10n.defaultFailedLoginDescription
+                        )
+                        TextPair(
+                            title: L10n.custom,
+                            subtitle: L10n.customFailedLoginDescription
+                        )
+                    }
+                }
+            }
+        }
+
+        // MARK: - Failed Login Selection Button
+
+        @ViewBuilder
+        private func MaxFailedLoginsButton() -> some View {
+            ChevronAlertButton(
+                L10n.customFailedLogins,
+                subtitle: Text(tempLoginAttempts ?? 1, format: .number),
+                description: L10n.enterCustomFailedLogins
+            ) {
+                let loginAttemptsBinding = $tempLoginAttempts
+                    .coalesce(1)
+                    .clamp(min: 1, max: 1000)
+
+                TextField(L10n.failedLogins, value: loginAttemptsBinding, format: .number)
+                    .keyboardType(.numberPad)
+
+            } onSave: {
+                policy.loginAttemptsBeforeLockout = tempLoginAttempts
+            } onCancel: {
+                tempLoginAttempts = policy.loginAttemptsBeforeLockout
+            }
+        }
+
+        // MARK: - Failed Login Validation
+
+        @ViewBuilder
+        private var MaxSessionsView: some View {
+            Section {
+                CaseIterablePicker(
+                    L10n.maximumSessionsPolicy,
+                    selection: $policy.maxActiveSessions.map(
+                        getter: { ActiveSessionsPolicy(rawValue: $0) ?? .custom },
+                        setter: { $0.rawValue }
+                    )
+                )
+                .onChange(of: policy.maxActiveSessions) { _ in
+                    tempMaxSessions = policy.maxActiveSessions
+                }
+
+                if policy.maxActiveSessions != ActiveSessionsPolicy.unlimited.rawValue {
+                    MaxSessionsButton()
+                }
+
+            } footer: {
+                VStack(alignment: .leading) {
+                    Text(L10n.maximumConnectionsDescription)
+                    LearnMoreButton(L10n.maximumSessionsPolicy) {
+                        TextPair(
```

Comment:
I believe this isn't relevant to this section?

Reply:
You are absolutely right. I mixed that up. I moved that to the FailedLoginsView where I meant to put that.